src/selectfield/selectfield.js

```diff
@@ -167,6 +167,11 @@
         this.select_.addEventListener('focus', this.boundFocusHandler);
         this.select_.addEventListener('blur', this.boundBlurHandler);
         this.select_.addEventListener('reset', this.boundResetHandler);
+        
+        var icon = document.createElement('div');
+        icon.className = 'mdl-selectfield__icon';
```

Comment:
I am not a big fan of creating elements like this, this should work by adding the element in the html directly, something like:
```html
<div class="mdl-selectfield mdl-js-selectfield">
  <select id="gender" class="mdl-selectfield__select">
    <option value=""></option>
    <option value="option1">option 1</option>
    <option value="option2">option 2</option>
  </select>
  <i class="material-icons">arrow_drop_down</i>
  <label class="mdl-selectfield__label" for="gender">User gender</label>
  <span class="mdl-selectfield__error">Select a value</span>
</div>
```